integration/acceptance/splitter_spec.rb

```diff
@@ -0,0 +1,30 @@
+# encoding: utf-8
+
+require File.expand_path("../../spec_helper", __FILE__)
+
+module Backup
+  describe Splitter do
```

Comment:
the nesting within the `Backup` module is something that has bothered me with other specs, it's not really necessary. We can also just write this directly:

```ruby
describe Backup::Splitter do
```

Reply:
I'll update this PR, and do another to clean up the existing files in the integration/ directory.

Thank you for reviewing!